.gitignore

```diff
@@ -91,3 +91,5 @@ app/coverage_html_report/
 
 # Dependencies
 node_modules
+
+google-credentials.json
```

Comment:
@emtwo I ended up going with google-credentials.json over credentials.json so it's a little less ambiguous.